src/frontend/components/ViewersList/components/ViewersListItemContainer/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box } from 'grommet';
```

Comment:
are tests missing for this component ?

Reply:
There is no test file associated to this component because it only makes  some layout. Due to the fact we decided to not use `imageSnapshot` anymore, IMO there is no relevant test which can be made here

Reply:
new test added in : [27cfa3](https://github.com/openfun/marsha/pull/1443/commits/27cfa3d31bff1be2923051bc8eeb508ab534558a)